app/Http/Controllers/CharacterController.php

```diff
@@ -28,7 +28,7 @@ public function __construct()
     private function getValidationRules() {

```

Comment:
`getValidationRules()` is called in the `updateNote()`. This was done at the time because there weren't any validation conflicts with reusing `getValidationRules()` in there. (`updateNote()` is called when editing the note on the page to [show a character](https://github.com/thatsmybis/thatsmybis/blob/master/resources/views/character/show.blade.php#L106))

With `name` now set to required, I think there's going to be a problem. Updating the notes on their own would now require a name to be set. You can test it to make sure, but I think you'll want to change the validation for `updateNote()` to only validate the necessary fields ([public_note, officer_note, id](https://github.com/thatsmybis/thatsmybis/blob/master/resources/views/character/show.blade.php#L106)).

Replacing `getValidationRules()` with this array would be fine enough:
```
[
     'id' => 'required|integer|exists:characters,id',
     'public_note'   => 'nullable|string|max:144',
     'officer_note'  => 'nullable|string|max:144',
];
```

Or, taking `name` out of `getValidationRules()` and only including it in the functions that need it would work too. This way we wouldn't have to copy+paste the note validation rules into multiple places, but it _does_ feel a little convoluted.

Reply:
It's honestly been seven years since I've actively commented on PR's or reviewed them. Let me know if that was too verbose.

Reply:
Nope, this is great feedback. I had tested the notes on everything except the character page. I took your advice and updated the the validation rules for updateNote().